python/myelin/lexical_encoder.py

```diff
@@ -25,7 +25,9 @@ def __init__(self, flow,  spec, lstm_feature_embeddings, lr_lstm, rl_lstm):
     lexicon.type = "dict"
     lexicon.add_attr("delimiter", 10)
     lexicon.add_attr("oov", spec.words.oov_index)
-    lexicon.add_attr("normalize_digits", spec.words.normalize_digits)
+    normalization = ""
+    if (spec.words.normalize_digits) normalization = "d"
+    lexicon.add_attr("normalization", normalization)
```

Comment:
As discussed over chat:
On the next line, we set lexicon.data to str(spec.words), which uses unnormalized form of words. Later in Lexicon::InitWords(), we expect normalized words, so this would cause a disparity.

Reply:
The Lexicon.add() method normalizes the words when they are inserted, so I believe that the Lexicon.index_to_item contains the normalized words. Am I missing something here?